tests/Feature/Filament/SpotifySyncPageTest.php

```
@@ -0,0 +1,200 @@
+        /* Arrange */
+        $mockTokenManager = Mockery::mock(SpotifyTokenManager::class);
+        $mockTokenManager->shouldReceive('isAuthenticated')->andReturn(true);
+        $this->app->instance(SpotifyTokenManager::class, $mockTokenManager);
+        $this->actingAs($this->user);
+
+        /* Act */
+        $response = Livewire::test(SpotifySync::class);
+
+        /* Assert */
+        $response->assertSee('Connected');
+    }
```

PHPMD (2.15.0)

The method it_shows_connected_status_when_authenticated is not named in camelCase. (undefined)

(CamelCaseMethodName)